<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MachineInfo</class>
 <widget class="QDialog" name="MachineInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Machine Information</string>
  </property>
  <widget class="QLineEdit" name="MaxFSLLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MaxFSLLabel">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>210</y>
     <width>270</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Length:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="MaxFSWLabel">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>190</y>
     <width>270</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="MLCLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>114</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>MLC Type:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ModelLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>93</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Model:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ManufacturerLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>72</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Manufacturer:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="FileNameLable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>51</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Machine File Name:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="HeadTextLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;font size=&quot;+1&quot;&gt;&lt;b&gt;Verify the Machine Information and then Save and Exit&lt;/b&gt;&lt;/font&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="FileNameLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ManufacturerLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>71</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ModelLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>92</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditMLCtype">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>113</y>
     <width>130</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMLCtype">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>114</y>
     <width>140</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SIMPLE-MLC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DBLFOCUS-MLC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RNDFOCUS-MLC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ELEKTA-MLC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VARIAN-MLC</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="MaxFSLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum Field Size (mm)</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="NormEnergyLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Nominal Energy (MV):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="EnergyLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="MaxFSWLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SFDLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Source to Flattening Filter Qt::DockBottom Distance (cm):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="SFDLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ESCDLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>271</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Effective Source to Collimator Distance (cm):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="iMLCLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>571</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Inner Thin MLCs:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LJTLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>351</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Lower Jaw Thickness (cm):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="tkMLCLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>551</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Thick MLC Width (mm):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cMLCLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>431</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Center of Circle of MLC Round Leaf (cm):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ESCDLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>271</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="thMLCLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>591</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Thin MLC Width (mm):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="rMLCLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>411</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Radius of Circle Center of MLC Round Leaf (cm):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="xMLCLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>491</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MLC Included in X Penumbra:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="SLJDLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>311</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="tkMLCLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>551</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SUJDLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>291</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Source to Upper Jaw Distance (cm):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="SMDLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>371</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Source to MLC (Middle) Distance (cm):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="cMLCLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>431</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="oMLCLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>451</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Qt::Orientation of MLC Traveling:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="tMLCLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>391</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MLC Thickness (cm):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="SUJDLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>291</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LJTLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>351</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="iMLCLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>571</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SLJDLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>311</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Source to Lower Jaw Distance (cm):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="UJTLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>331</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nMLCLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>531</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="xMLCcheckBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>491</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="thMLCLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>591</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="yMLCLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>511</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MLC Included in Y Penumbra:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="UJTLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>331</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Upper Jaw Thickness (cm):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="yMLCcheckBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>510</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="SMDLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>371</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="rMLCLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>411</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="tMLCLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>391</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="oMLCcomboBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>451</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>X</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Y</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="isMLCLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>471</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MLC Included in Model:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nMLCLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>531</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Number of Leaf Pairs:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>623</y>
     <width>110</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>&amp;Reset to XiO</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>623</y>
     <width>110</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>&amp;Save and Done</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="interruptButton">
   <property name="geometry">
    <rect>
     <x>318</x>
     <y>623</y>
     <width>110</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>&amp;Cancel or Skip</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </widget>
  <widget class="QLabel" name="AvalLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Flattening Filter Attenuation Factor (A-VALUE):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="AvalLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>9.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>EnergyLineEdit</tabstop>
  <tabstop>MaxFSWLineEdit</tabstop>
  <tabstop>MaxFSLLineEdit</tabstop>
  <tabstop>SFDLineEdit</tabstop>
  <tabstop>ESCDLineEdit</tabstop>
  <tabstop>SUJDLineEdit</tabstop>
  <tabstop>SLJDLineEdit</tabstop>
  <tabstop>UJTLineEdit</tabstop>
  <tabstop>LJTLineEdit</tabstop>
  <tabstop>SMDLineEdit</tabstop>
  <tabstop>tMLCLineEdit</tabstop>
  <tabstop>rMLCLineEdit</tabstop>
  <tabstop>cMLCLineEdit</tabstop>
  <tabstop>oMLCcomboBox</tabstop>
  <tabstop>xMLCcheckBox</tabstop>
  <tabstop>yMLCcheckBox</tabstop>
  <tabstop>nMLCLineEdit</tabstop>
  <tabstop>tkMLCLineEdit</tabstop>
  <tabstop>iMLCLineEdit</tabstop>
  <tabstop>thMLCLineEdit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>interruptButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>xMLCcheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>xMLCcheckBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yMLCcheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>yMLCcheckBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oMLCcomboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>oMLCcomboBox</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>MachineInfo</receiver>
   <slot>checkModified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>interruptButton</sender>
   <signal>clicked()</signal>
   <receiver>MachineInfo</receiver>
   <slot>Cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>MachineInfo</receiver>
   <slot>resetMachine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
